Exports/_export/_modules/custom-type/products.php

```diff
@@ -12,115 +12,121 @@
 
 	require('../../_inc/config.php');
 
-	$filename = getSiteId() . '_' . 'products' . 'Export' . date('M') . '_' . date('d') . '_' . date('Y');
-	$howmany = 100; // Set to number of products in the module
-
-	// Header
-	header("Content-type: text/csv");
-	header("Content-Disposition: attachment; filename=" . $filename . ".csv");
-	header("Pragma: no-cache");
-	header("Expires: 0");
-
+	set_time_limit(0);
+	
 	// Functions
-	function processItem($in){
+	function processItem($in)
+	{
 		$out = trim($in);
 		$out = str_replace('"','""',$out);
 		$out = str_replace('&amp;','&',$out);
 		$out = '"' . $out . '"';
+
 		return $out;
 	}
 
+	$filename = getSiteId() . '_' . 'products' . 'Export' . date('M') . '_' . date('d') . '_' . date('Y');
+	$howmany = 10000; // Set to number of products in the module
+
+	// Header
```

Comment:
_headers_ only need to be set once, but when they are inside of the `processItem` function they will get set multiple times.  I would put the headers back where they were previously, or right before you echo out the content near the bottom of the page.